src/extensions/sessionrecording.js

```diff
@@ -24,6 +25,18 @@ export class SessionRecording {
         }
     }
 
+    started() {
+        return this.captureStarted
+    }
+
+    stopRecording() {
+        if (this.captureStarted && this.stopRrweb) {
```

Comment:
why is this different from `this.started()`? Makes things harder to reason about.

Reply:
This is me writing as I would TS. If `captureStarted` it should be there, but TS would have screamed at me saying this is possibly null